src/components/CustomizedPipeline/CustomizePipelines.tsx

```diff
@@ -48,53 +37,6 @@ type Props = RawComponentProps & {
   singleComponent?: boolean;
 };
 
-const ComponentKebab: React.FC<
-  React.PropsWithChildren<{
-    state: PACState;
-    component: ComponentKind;
-    canPatchComponent?: boolean;
-  }>
-> = ({ component, state, canPatchComponent }) => {
-  const namespace = useNamespace();
-  const track = useTrackEvent();
-  const [isOpen, setOpen] = React.useState(false);
-  return (
-    <Dropdown
-      onSelect={() => setOpen(false)}
-      toggle={<KebabToggle onToggle={() => setOpen((v) => !v)} id="toggle-id-6" />}
-      isOpen={isOpen}
-      isPlain
-      position={DropdownPosition.right}
-      dropdownItems={[
-        <DropdownItem
-          key="roll-back"
-          isDisabled={![PACState.error, PACState.pending, PACState.ready].includes(state)}
-          onClick={() => {
-            track(TrackEvents.ButtonClicked, {
-              link_name: 'disable-pac',
-              link_location: 'manage-builds-pipelines-action',
-              component_name: component.metadata.name,
-              app_name: component.spec.application,
-              namespace,
-            });
-            void disablePAC(component).then(() => {
```

Comment:
@janaki29 Is there any other use of `disablePAC` in the codebase. I dint find any?? If not then should we have it removed as well? @sahil143 

PR looks good LGTM